lms/envs/common.py

```diff
@@ -2170,8 +2170,19 @@
     'terms_of_service': 'hidden',
     'city': 'hidden',
     'country': 'hidden',
+    'dropdown': 'hidden',
 }
 
+REGISTRATION_DROPDOWN_LABEL = 'How did you hear of us?'
```

Comment:
I think it would be better to move the default values (this and the next two settings) into the Django view code, so that they can be marked for translation (and then also you don't need to duplicate them between CMS and LMS).